pythonbible/regular_expressions.py

```diff
@@ -152,7 +152,7 @@ def _add_suffix(regex: str, suffix: str = None) -> str:
     ),
     Book.TITUS: r"Tit\.*(?:us)?",
     # assume 'Phi' is Philemon if Philippians failed?
-    Book.PHILEMON: r"(Phi|Phlm|Phile)\.*(?:m\.*(?:on)?)?", 
+    Book.PHILEMON: r"(Philemon|Phile|Phi|Phlm).(?:m.(?:on)?)?", 
```

Comment:
Maybe a better, or at least more readable, regex for Philemon would be:

Book.PHILEMON: r"(Philemon|Philem\.*|Phile\.*|Phlm\.*|Phi\.*)",

We've got one test currently failing because of the "."

Reply:
I think the formatting of my previous review messed up the regular expression that I was recommending. Sorry about that.

Reply:
NP, I went ahead and committed your comment, awaiting the test suite. Seems nice to be more explicit for this edge-case, anyway.